### PR TITLE
Dansholds/cyber sec content

### PR DESCRIPTION
**Pull request by dansholds**
transformed all content to current styling & completed topics that had 0 content.

### CODE REVIEW

File: src/data/roadmaps/cyber-security/content/aws@0LztOTc3NG3OujCVwlcVU.md [hunk 1]
```diff
@@ -4,4 +4,8 @@ Amazon Web Services (AWS) is a leading cloud computing platform provided by Amaz
 
 Learn more from the following resources:
 
+- [@roadmap@AWS Roadmap](/aws)
```
Comment (kamranahmedse) [inline src/data/roadmaps/cyber-security/content/aws@0LztOTc3NG3OujCVwlcVU.md:7]: These roadmap URLs should be full URLs.

Comment (dansholds) [reply src/data/roadmaps/cyber-security/content/aws@0LztOTc3NG3OujCVwlcVU.md:7]: Ok, it isn't in some roadmaps so we'll have to go and change it in all locations.

File: src/data/roadmaps/cyber-security/content/ssl--tls@LKK1A5-xawA7yCIAWHS8P.md [hunk 2]
```diff
@@ -1,37 +1,8 @@
...
-
-- **Authentication:** SSL/TLS uses digital certificates to authenticate the server and sometimes the client. This helps to ensure that the parties involved in the communication are who they claim to be.
-
-- **Data integrity:** SSL/TLS includes mechanisms to confirm that the data received has not been tampered with during transmission, maintaining the integrity of the information being sent.
+Single Sign-On (SSO) is an authentication method that allows users to access multiple applications or systems with one set of login credentials. It enables users to log in once and gain access to various connected systems without re-entering credentials. SSO enhances user experience by reducing password fatigue, streamlines access management for IT departments, and can improve security by centralizing authentication controls. It typically uses protocols like SAML, OAuth, or OpenID Connect to securely share authentication information across different domains. While SSO offers convenience and can strengthen security when implemented correctly, it also presents a single point of failure if compromised, making robust security measures for the SSO system critical.
```
Comment (shofer) [inline src/data/roadmaps/cyber-security/content/ssl--tls@LKK1A5-xawA7yCIAWHS8P.md:3]: @dansholds Hi, i guess something got mixed here, there a several occasions where the headline says SSL / TLS but the content is actually about SSO

File: src/data/roadmaps/cyber-security/content/ssl-and-tls-basics@dJ0NUsODFhk52W2zZxoPh.md [hunk 3]
```diff
@@ -1,33 +1,8 @@
...
-- **Secured connection**: Once the keys are exchanged, the client and server can now communicate securely using the established encryption and keys.
-
-Secure communication is critical for any organization handling sensitive data. SSL and TLS serve as the backbone for protecting data in transit and play a significant role in ensuring the confidentiality, integrity, and authenticity of online communications.
+Single Sign-On (SSO) is an authentication method that allows users to access multiple applications or systems with one set of login credentials. It enables users to log in once and gain access to various connected systems without re-entering credentials. SSO enhances user experience by reducing password fatigue, streamlines access management for IT departments, and can improve security by centralizing authentication controls. It typically uses protocols like SAML, OAuth, or OpenID Connect to securely share authentication information across different domains. While SSO offers convenience and can strengthen security when implemented correctly, it also presents a single point of failure if compromised, making robust security measures for the SSO system critical.
```
Comment (shofer) [inline src/data/roadmaps/cyber-security/content/ssl-and-tls-basics@dJ0NUsODFhk52W2zZxoPh.md:3]: @dansholds same here SSL/TLS -> SSO

File: src/data/roadmaps/cyber-security/content/ssl-vs-tls@6ILPXeUDDmmYRiA_gNTSr.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# SSL vs TLS
+
+Single Sign-On (SSO) is an authentication method that allows users to access multiple applications or systems with one set of login credentials. It enables users to log in once and gain access to various connected systems without re-entering credentials. SSO enhances user experience by reducing password fatigue, streamlines access management for IT departments, and can improve security by centralizing authentication controls. It typically uses protocols like SAML, OAuth, or OpenID Connect to securely share authentication information across different domains. While SSO offers convenience and can strengthen security when implemented correctly, it also presents a single point of failure if compromised, making robust security measures for the SSO system critical.
```
Comment (shofer) [inline src/data/roadmaps/cyber-security/content/ssl-vs-tls@6ILPXeUDDmmYRiA_gNTSr.md:3]: @dansholds same and here SSL/TLS -> SSO

**Pull request merged**